docs/consume-packages/Package-Restore.md

```diff
@@ -115,6 +115,17 @@ To enable or disable Package Restore for all users on a computer, a developer or
 > [!Important]
 > If you edit the `packageRestore` settings directly in `nuget.config`, restart Visual Studio, so that the **Options** dialog box shows the current values.
 
+### Choose default package management format
+
+![Control default package management format though NuGet Package Manager options](media/Restore-02-PackageFormatOptions.png)
+
+NuGet has two formats in which a project may use packages: [`PackageReference`](package-references-in-project-files.md) and [`packages.config`](../reference/packages-config.md). The default format can be selected from the drop-down under the **Package Management** heading. An option to be prompted when the first package is installed in a project is also available.
+
+> [!Note]
+> If a project does not support both package management formats, the package management format used will be the one that's compatible with the project, and therefore may not be the default set in the options. Additionally, NuGet will not prompt for selection on first package installation, even if the option is selected in the options window.
+>
+> If Package Manager Console is used to install the first package in a project, NuGet will not prompt for format selection, even if the option is selected in the options window.
```

Comment:
Wait, which format does it use in this case?

Reply:
It uses the selected drop-down value as usual.